### ISSUE TITLE
How do you train custom corpus with bert?

### ISSUE THREAD

**Issue by shadylpstan** [huggingface/pytorch-pretrained-BERT#443]
I am using a domain specific dataset for text classification.
But major of my data points are treated with [UNK] token in Bert.

Can I please get help on how to keep my custom corpus tokens?

**Comment by thomwolf** [comment]
Maybe it's a problem of not setting `do_lower_case=False` and using a `cased` model like in #436.
Which pretrained model are you using?

**Comment by shadylpstan** [comment]
@thomwolf 
sorry but this is completely not what I am asking for.
Suppose I have medical data and the names of the medicines and diseases are not in the bert pre-trained corpus. So bert gives me [UNK] token for important medical terms. How can I train it on my custom data?

**PS: Medical data was just an example.**

**Comment by nikhilvishwanath** [comment]
Hi, I'm also facing similar issue plz let me know if you have found any solution @shadylpstan

**Comment by shadylpstan** [comment]
> Hi, I'm also facing similar issue plz let me know if you have found any solution @shadylpstan

I was successful in modifying a couple of bert files and train it using my corpus.
Though it was really a mess doing so because by default bert has a classification model of a certain number of classes.
The accuracy wasn't good enough. So no idea it is a worth giving it shot from your end.
